Localize/lang/strings.json

```diff
@@ -163,11 +160,11 @@
   "4wjJs0": "14",
   "5+MuPz": "Enter a valid double.",
   "5FINOY": "Workflow has parameter validation errors on the following operations: {invalidNodes}",
-  "5HGtu7": "Function removed from current location. Function exists elsewhere.",
   "5J7j9k": "Required. The XPath expression to evaluate.",
   "5MWmiO": "Line Number",
   "5OvGgn": "Body",
   "5SAQOb": "Authority",
+  "5UPOY+": "Select this if you are configuring an on-premises connection.",
```

Comment:
Unclear what "this" references? A checkbox?

Reply:
Yes, a checkbox